asset/gulpfile.js

```diff
@@ -40,39 +37,65 @@ var gulp = require('gulp'),
         'yaml'
     ];
 
+// Images
 gulp.task('images', function () {
     return gulp.src('img/*')
         .pipe(gulp.dest('../theme/img/'));
 });
 
+// Icons
+gulp.task('icons', function () {
+    return gulp.src('node_modules/font-awesome/fonts/**.*')
+        .pipe(gulp.dest('../theme/css/fonts'));
+});
+
+// JavaScript
 gulp.task('scripts', function () {
     var prismComponents = [];
     for (var component in prism) {
-        prismComponents[component] = 'node_modules/prismjs/components/prism-' + prism[component] + '.min.js';
+        prismComponents[component] = 'node_modules/prismjs/components/prism-' + prism[component] + '.js';
     }
 
-    return gulp.src(prismComponents.concat([
-            'node_modules/prismjs/plugins/normalize-whitespace/prism-normalize-whitespace.min.js',
-            'node_modules/anchor-js/anchor.min.js',
-            'js/base.js',
-            'js/scripts.js'
-        ]))
-        .pipe(concat({path: 'scripts.js'}))
-        .pipe(uglify({mangle: false}))
+    return gulp.src(
+        prismComponents.concat(
+            [
+                'node_modules/prismjs/plugins/normalize-whitespace/prism-normalize-whitespace.min.js',
+                'node_modules/jquery/dist/jquery.min.js',
+                'node_modules/bootstrap-sass/assets/javascripts/bootstrap.min.js',
+                'node_modules/anchor-js/anchor.min.js',
+                'js/base.js',
+                'js/component-list.js'
+            ]
+        )
+    )
```

Comment:
I don't like this formatting, next line is intended one more, it's confusing. imho the previous one was better.

Reply:
In my opinion is this form of formatting not a topic for this repository. We have no coding standard for this file type and we have other, bigger problems to solve.

The focus should be on the user / reader of the documentation and not the three or five developers of this template.